transcom/mymove
# B 20250 incorrect gbloc prime int

## [Agility ticket](tbd)

## Summary

This PR updates the originDutyLocationGBLOC to show USMC for marine orders.
This is  a non breaking change to the Prime so all versions of the prime has been updated with the same change.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create a  HHG move and set your service branch as "Marine Corps".
3. Login as a SC to submit move. Edit the department indicator to show "17 Navy and Marine Corps"
4. Login as TOO in USMC GBLOC and approve and send move
5. Login as Prime in USMC GBLOC and find move using move code
6. After opening move as prime:
     6a. Open developer tools 
     6b. Go to moveTaskOrders endpoint and look under preview tab.
     6c. Click dropdown arrow next to "order" 
     6d. Confirm that "originDutyLocationGBLOC" shows USMC

### Backend


## Screenshots
Before:
<img width="398" alt="Screenshot 2024-06-20 at 11 20 44 AM" src="https://github.com/transcom/mymove/assets/146969726/97cd0856-6294-4cef-bd5c-3618cd7dfbf9">

After:
<img width="285" alt="Screenshot 2024-06-20 at 11 21 26 AM" src="https://github.com/transcom/mymove/assets/146969726/5e0ef58d-a270-4e07-ae31-e28045558148">
